preprocessing/lib/csvToJson.js

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
for consistency, please move all camelCased file names to dashed-names, fix dependencies

Reply:
Resolved in 2f993202d2d345103db80633f8bfd4abdd42bd9e